services/ingest.json

```diff
@@ -1,894 +1,930 @@
 {
-  "timestamp": "2018-08-02T17:52:35.469Z",
+  "timestamp": "2018-08-04T15:18:27.072Z",
   "core": {
-    "url": "http://mirrors.jenkins.io/war/latest/jenkins.war",
+    "url": "http://mirrors.jenkins.io/war/2.135/jenkins.war",
     "checksum": {
-      "type": "sha256",
-      "signature": "latest"
+      "signature": "d682c43b381898fde9ace47892bdf9277ed2389f15299356e20e1a85720cdeb0",
+      "type": "sha256"
     }
   },
   "plugins": [
     {
-      "groupId": "org.jenkins-ci.plugins",
+      "groupId": "io.jenkins",
       "artifactId": "configuration-as-code",
       "url": "https://repo.jenkins-ci.org/incrementals/io/jenkins/configuration-as-code/0.11-alpha-rc362.942711740b07/configuration-as-code-0.11-alpha-rc362.942711740b07.hpi",
       "checksum": {
-        "type": "sha512",
-        "signature": "4faefcc46299557efaa1c896835a2413ad0646201e40938b4207dbd4cfecf33303cf5f28fd1f53cb71146c45d12c7fa34d2d87d312acf9fd7964c08506d2278d"
+        "type": "sha256",
+        "signature": "5e0d07f6bc86ea23dbe6f2b40d15fe8759aa43470a45d9ee09d90f415e007575"
       }
     },
     {
       "groupId": "io.jenkins.plugins",
       "artifactId": "essentials",
-      "url": "http://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/essentials/0.3-rc20.a3f76cdd1348/essentials-0.3-rc20.a3f76cdd1348.hpi",
+      "url": "https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/essentials/0.3-rc20.a3f76cdd1348/essentials-0.3-rc20.a3f76cdd1348.hpi",
       "checksum": {
-        "type": "sha512",
-        "signature": "e1cdf271eabed428c2198c4f7a49c5dea7fc8850987bcf63b87b74936a3cc426896a4b7c9c974f0fe3abd61071ea5a2238538af9f9266f2d37696856809d697a"
+        "type": "sha256",
+        "signature": "392f0a30de31dadcaa2a06b8e07ff721bcf8fe37eb520d5a669dd5a8cdb17301"
       }
     },
     {
       "groupId": "io.jenkins.blueocean",
       "artifactId": "blueocean",
-      "url": "http://updates.jenkins-ci.org/download/plugins/blueocean/1.7.1/blueocean.hpi",
+      "url": "https://repo.jenkins-ci.org/releases/io/jenkins/blueocean/blueocean/1.7.1/blueocean-1.7.1.hpi",
```

Comment:
This looks like a mistake.

Reply:
Sorry, was on mobile.

You are switching from a (presumably scalable) Jenkins-run official update site to an Artifactory repository, for a release version of a plugin. IMO there should be no `url`s using https://repo.jenkins-ci.org/releases/ since these could all be served by http://updates.jenkins-ci.org/download/plugins/ instead.